pkg/apis/core/types.go

```diff
@@ -5350,6 +5350,13 @@ type WindowsSecurityContextOptions struct {
 	// PodSecurityContext, the value specified in SecurityContext takes precedence.
 	// +optional
 	RunAsUserName *string
+
+	// HostProcess determines if a container should be run as a 'Host Process' container.
+	// This field is alpha-level and will only be honored by components that enable the
+	// WindowsHostProcessContainers feature flag. Setting this field without the feature
+	// flag will result errors when validating the Pod.
```

Comment:
This should also describe the requirement that all containers effective `hostProcess` value must agree, and that if a pod is `hostProcess`, it must also set `hostNetwork` to true

Reply:
I've updated the wording to reflect this requirement